tastyworks/dxfeed/mapped_item.py

```diff
@@ -55,3 +55,56 @@ def __init__(self, data=None):
         if data:
             self.data = self._map_data(data)
         self.keys = None
+
```

Comment:
Same as above, this is just some notes that may come in handy in the future, maybe better in a separate documentation folder?